.github/workflows/test.yaml

```diff
@@ -0,0 +1,20 @@
+name: Go test
```

Comment:
Can you lint the file? it's hard to read as it's with out any space between sections.